app/views/comments/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<div>
+  <h1>New Post</h1>
+  <%= form_with model: @new_comment, url: user_post_comments_path(@current_user) do |f| %>
+    <div>
+      <%= f.label :text %><br>
+      <%= f.text_field :text %><br>
+    </div>
+    <div>
+      <%= f.submit "post" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Good job adding a form to create a comment, however, currently we are not able to add a comment. Kindly fix it 👍 
- [ ] [OPTIONALY] I could be better to change the text in your `<h1>New Post</h1>` because we are not creating a post

![Capture](https://github.com/CobbyElsonfx/blog_rift/assets/57408419/57df5d03-de50-4436-b937-64d5df9ef27a)